client/java/src/main/java/com/linecorp/centraldogma/client/CentralDogma.java

```diff
@@ -615,7 +616,7 @@ default CompletableFuture<Revision> watchRepository(String projectName, String r
      *
      * @return the latest known {@link Revision} which contains the changes for the matched files.
      *         {@code null} if the files were not changed for {@code timeoutMillis} milliseconds
-     *         since the invocation of this method.
+     *         since the invocation of this method, or the server is shut down during the watch.
```

Comment:
Instead of saying that the server is shutting down, why don't we focus on that a watch can be finished before the timeout? For example:
```
Even before the timeout, the watch may return null earlier due to issues such as server restart.
```